dqvm/src/utils.lisp

```diff
@@ -0,0 +1,37 @@
+;;;; src/utils.lisp
+;;;;
+;;;; Author: Juan M. Bello-Rivas
+;;;;         Robert Smith
+;;;;         Lauren Capelluto
+
+(in-package #:dqvm2)
+
+(defun error-missing-initform (symbol)
+  (error "You must specify ~a" (prin1-to-string symbol)))
```

Comment:
`prin1-to-string` needed here?

Reply:
Not anymore. The rationale was the same as in my comment regarding `print-object`. Thanks for pointing it out.